modules/openapi-generator/src/main/resources/kotlin-misk/apiAction.mustache

```diff
@@ -38,14 +37,14 @@ import misk.web.mediatype.MediaTypes
 {{#operations}}
 @Singleton
 class {{classname}}Action @Inject constructor(
-) : WebAction {
+) : {{actionParentClass}} {
 {{#operation}}
 
-    @{{httpMethod}}("{{actionPathPrefix}}{{path}}")
+    @{{httpMethod}}("{{{actionPathPrefix}}}{{path}}")
     @Description("{{{summary}}}"){{#hasConsumes}}
     @RequestContentType({{#consumes}}{{{mediaType}}}{{^-last}}, {{/-last}}{{/consumes}}){{/hasConsumes}}{{#hasProduces}}
-    @ResponseContentType({{#produces}}{{{mediaType}}}{{^-last}}, {{/-last}}{{/produces}}){{/hasProduces}}
-    @LogRequestResponse(bodySampling = 1.0, errorBodySampling = 1.0)
+    @ResponseContentType({{#produces}}{{{mediaType}}}{{^-last}}, {{/-last}}{{/produces}}){{/hasProduces}}{{#actionAnnotations}}
+    {{{.}}}{{/actionAnnotations}}
```

Comment:
```
    @ResponseContentType({{#produces}}{{{mediaType}}}{{^-last}}, {{/-last}}{{/produces}}){{/hasProduces}} 
    {{#actionAnnotations}}
    {{{.}}}
    {{/actionAnnotations}}
```
please put start and end tags in separate lines to avoid blank lines